tests/system/feature_store/test_spark_engine.py

```diff
@@ -116,6 +116,14 @@ def get_remote_pq_target_path(self, without_prefix=False):
         path += "/bigdata/" + self.pq_target
         return path
 
+    @staticmethod
+    def get_remote_target_dir_path(without_prefix=False):
+        path = "v3io://"
+        if without_prefix:
```

Comment:
Why we need this param? Does spark will work without a prefix of v3io, az, or s3?

Reply:
I just made it in line with the rest of the test suite. If I remove it, should probably remove it everywhere.